### PR TITLE
Replace tox with raw calls to pytest

### PR DESCRIPTION
**Pull request by wlach**
Tox has a really long setup cycle, and we don't really need its functionality
for testing missioncontrol (which is a web application running in a deterministic
environment)

### CODE REVIEW

**Comment by wlach** [comment]
Hey Rob, would you mind taking a look at this one? It came out of me being frustrated with the insanely long setup times I had with tox. Basically this moves missioncontrol to using a similar testing setup to atmo.

The only somewhat silly thing about this is that it currently installs the testing dependencies in the production container by default. We were already doing this to some extent before, so it doesn't seem like a big regression.

File: requirements/build.txt [hunk 1]
```diff
@@ -167,63 +151,15 @@ docutils==0.14 \
...
-    --hash=sha256:9824e15b387d331c0fc0fef905a539ab69784368a1d6ac3db864b4182e520948 \
-    --hash=sha256:4a678e1b9619a29c51301af61ab84122e2f8cc7a0a6b40854b808ac6be604300 \
-    --hash=sha256:8bb7c8dca54109b61013bc4114d96effbf10dea136722c586bce3a5d9fc4e730 \
-    --hash=sha256:1a41d621aa9b6ab6457b557a754d50aaff0813fad3453434de075496fca8a183 \
-    --hash=sha256:0fa423599fc3d9e18177f913552cdb34a8d9ad33efcf52a98c9d4b644edb42c5 \
-    --hash=sha256:e61a4ba0b2686040cb4828297c7e37bcaf3a1a1c0bc0dbe46cc789dde51a80fa \
-    --hash=sha256:ce9ef0fc99d11d418662e36fd8de6d71b19ec87c2eab961a117cc9d087576e72
+django-configurations==2.0 \
+    --hash=sha256:b4a4eb3ed631c2abbeb7008b5cc5d8d04b190df838e7b613731d728f674f3a2f \
+--hash=sha256:be47e25d070fd005cc8fbd6c913c9ec37e83d04686937afe18ca6c579b2cdedb
```
Comment (robhudson) [inline requirements/build.txt]: this line isn't indented like the others

File: requirements/tests.txt [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    --hash=sha256:6d5bd4f7113b444c55a3bbb5c738a3dd80d43563d063fc42dcb0aaefbdd78b81
+pytest-cov==2.5.1 \
+    --hash=sha256:03aa752cf11db41d281ea1d807d954c4eda35cfa1b21d6971966cc041bbf6e2d \
+    --hash=sha256:890fe5565400902b0c78b5357004aab1c814115894f4f21370e2433256a3eeec
+pytest-django==3.1.2 \
+    --hash=sha256:038ccc5a9daa1b1b0eb739ab7dce54e495811eca5ea3af4815a2a3ac45152309 \
+    --hash=sha256:00995c2999b884a38ae9cd30a8c00ed32b3d38c1041250ea84caf18085589662
+pytest-flake8==0.9.1 \
+    --hash=sha256:e67686645860009cf1f9a5016e110234b4a2c4584711d62ba753ed651ab9ab28 \
+--hash=sha256:e716072d07a557defdd5c4141984569731e292961370a5663c1697283aa16200
```
Comment (robhudson) [inline requirements/tests.txt]: another one

File: requirements/build.txt (hunk 1 above)
Comment (wlach) [reply requirements/build.txt]: I think these were artifacts from copypasting some stuff from atmo :)

**Comment by wlach** [comment]
@robhudson Fixed nits, are we ok to land?

**Pull request merged**